Assets/Scripts/Models/Time/TimeManager.cs

```diff
@@ -198,6 +212,15 @@ public void Destroy()
     }
 
     /// <summary>
+    /// Runs the next frame before any updates are ran.
```

Comment:
Yeah that makes sense... I left that part as the autofilled summary tag had it... My brain was incorrectly applying it to the nextFrameActions field, rather than the function itself...